Sources/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift

```diff
@@ -55,6 +55,7 @@ public class SubscriptionPeriod: NSObject {
         }
 
         return .init(value: sk1SubscriptionPeriod.numberOfUnits, unit: unit)
+            .normalized()
```

Comment:
We can't test SK2's because it has no initializers, but maybe add a test in `SubscriptionPeriodTests` that ensures this is called here? It doesn't have to be comprehensive at all since you have all those examples in `StoreKitWorkaroundsTests`, maybe just one.

Reply:
Oooh, added in one with a few different scenarios to test. Great idea!